content/site-settings.yml

```diff
@@ -0,0 +1,5 @@
+instagram_url: 'https://www.instagram.com/distributeaid'
```

Comment:
This file contains global settings that can be applied on every page of the site.